packages/admin/test/e2e/cli.feature.ts

```diff
@@ -0,0 +1,515 @@
+          ['--destination', destinationWallet.address]
+        )
+      ).toThrowError('VM Exception while processing transaction: revert Caller not requester admin');
+    });
+  });
+
+  it('can set/get airnode parameters', async () => {
+    const airnodeWallet = ethers.Wallet.createRandom().connect(provider);
+    const airnodeMnemonic = airnodeWallet.mnemonic.phrase;
+    const hdNode = ethers.utils.HDNode.fromMnemonic(airnodeMnemonic);
+    const xpub = hdNode.neuter().extendedKey;
+    const masterWallet = new ethers.Wallet(hdNode.privateKey, provider);
+    // Fund the master wallet - which will be used to set the airnode parameters
+    await deployer.sendTransaction({
+      to: masterWallet.address,
+      value: ethers.utils.parseEther('1'),
+    });
+
+    const setAirnodeParametersOut = execCommand(
+      'set-airnode-parameters',
+      ['--mnemonic', airnodeMnemonic],
+      ['--derivationPath', 'm'],
+      ['--providerUrl', PROVIDER_URL],
+      ['--airnodeRrp', airnodeRrp.address],
+      ['--airnodeAdmin', bob.address],
+      ['--authorizersFilePath', `${__dirname}/../fixtures/authorizers.json`]
+    );
+    expect(setAirnodeParametersOut).toMatch(new RegExp('Airnode id: \\w+'));
+
+    const airnodeId = setAirnodeParametersOut.split('Airnode id: ')[1];
+    const getAirnodeParametersOut = execCommand(
+      'get-airnode-parameters',
+      ['--providerUrl', PROVIDER_URL],
+      ['--airnodeRrp', airnodeRrp.address],
+      ['--airnodeId', airnodeId]
+    );
+
+    const json = JSON.parse(getAirnodeParametersOut);
+    expect(json).toEqual({
+      admin: bob.address,
+      authorizers: ['0x0000000000000000000000000000000000000000', '0x0000000000000000000000000000000000000123'],
+      xpub,
+      blockNumber: expect.anything(),
+    });
+  });
+
+  it('derives endpoint id', () => {
+    const oisTitle = 'title';
+    const endpointName = 'endpoint';
+
+    const out = execCommand('derive-endpoint-id', ['--oisTitle', oisTitle], ['--endpointName', endpointName]);
+
+    const expected = ethers.utils.keccak256(
+      ethers.utils.defaultAbiCoder.encode(['string'], [`${oisTitle}_${endpointName}`])
+    );
+    expect(out).toBe(`Endpoint id: ${expected}`);
+  });
+
+  it('gets requester admin', async () => {
+    const requesterIndex = await admin.createRequester(airnodeRrp, alice.address);
+
+    const out = execCommand(
+      'requester-index-to-admin',
+      ['--providerUrl', PROVIDER_URL],
+      ['--requesterIndex', requesterIndex],
+      ['--airnodeRrp', airnodeRrp.address]
+    );
+
+    expect(out).toBe(`Requester admin: ${alice.address}`);
+  });
+
+  describe('incorrect usage', () => {
+    it('is missing command parameter', () => {
+      expect(() =>
+        execCommand(
+          'create-requester',
+          ['--mnemonic', mnemonic],
+          ['--providerUrl', PROVIDER_URL],
+          ['--airnodeRrp', airnodeRrp.address]
+          // missing --requesterAdmin parameter
```

Comment:
it explains that the `--requesterAdmin` is missing and that's the reason why the test should fail. 

I thought it will be easier to understand with comment, but I am not sure now :D

Reply:
Ah sorry my mistake. I didn't read the test properly